gxapi_rails.gemspec

```diff
@@ -25,16 +25,17 @@ Gem::Specification.new do |s|
   s.add_dependency(%q<rest-client>, [">= 0"])
   s.add_dependency(%q<json>, [">= 0"])
   s.add_dependency(%q<google-api-client>, [">= 0"])
+  s.add_dependency(%q<test-unit>, [">= 0"])
```

Comment:
Wasn't able to find it, but tests wouldn't run until I added it.  I believe it was implicitly included in activesupport, but then later removed.  Yay for dependency management.